packages/edit-site/src/components/global-styles/shadows-edit-panel.js

```diff
@@ -392,7 +392,13 @@ function ShadowItem( { shadow, onChange, canRemove, onRemove } ) {
 					'aria-expanded': isOpen,
 				};
 				const removeButtonProps = {
-					onClick: onRemove,
+					onClick: ( e ) => {
+						e.stopPropagation();
+						if ( isOpen ) {
+							onToggle();
+						}
+						onRemove();
+					},
```

Comment:
```suggestion
					onClick: () => {
						if ( isOpen ) {
							onToggle();
						}
						onRemove();
					},
```

What is `e.stopPropagation();` for? From my testing,g it seems to work without it 🤔

Reply:
I mixed up with another piece of the code and thought the buttons were nested, my bad. I've just removed this.